### ISSUE TITLE
Feature - Add possibility to customize error messages

### ISSUE THREAD

**Issue by korzio** [korzio/djv#16]
- Execute user callback when error is thrown. 

```
djv.setErrorMessage = function(errorType, dataType, ...){
....
}
```

- Add an option to show all messages (now it will stop to execute if any validator has failed)

**Comment by Anthropic** [comment]
It would be great if there was a way to provide the same error codes as TV4, our framework has relied on that for a looooong time and I want to get something faster embedded without causing a world of hurt for users. Just thought I'd mention it.

**Comment by korzio** [comment]
I'm looking into ajv implementation as well - it make sense to do it as close as possible between projects

**Comment by korzio** [comment]
Hey @Anthropic I've just created [a mr](https://github.com/korzio/djv/pull/28/files) regarding this feature. Please take a look into [a error handler format](https://github.com/korzio/djv/pull/28/files#diff-8e36e6270e46cbd86800680c612d2842R234) if you are still interested - now it is possible to generate any type of error, also with multi errors mode. This feature will be included in 1.1.0

**Comment by korzio** [comment]
Done https://github.com/korzio/djv/pull/28/files
